debug/sample.html

```diff
@@ -28,36 +31,46 @@
     <div id="map"></div>
 
     <script>
-      var map = L.map('map').setView([37.74, -121.62], 9);
+      var map = L.map("map").setView([37.74, -121.62], 9);
       // var tiles = L.esri.basemapLayer('Topographic').addTo(map);
-      L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-        attribution: '&copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors'
+      L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
+        attribution:
+          '&copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors'
       }).addTo(map);
 
       var searchControl = L.esri.Geocoding.geosearch({
+        position: "topright",
+        collapseAfterResult: true,
+        expanded: false,
+        zoomToResult: true,
+        useMapBounds: true,
```

Comment:
Also, one small change - could you please review these extra properties you added to the sample page?